src/sentry/eventtypes/error.py

```diff
@@ -22,42 +23,44 @@ def get_crash_location(data):
 
         func = get_function_name_for_frame(frame, data.get("platform"))
         return frame.get("filename") or frame.get("abs_path"), func
+    return None
+
+
+NATIVE_MOBILE_PLATFORMS = (
```

Comment:
nit: we really should put these types of constants in one well defined place, we keep on redefining them in bunch of places
 https://github.com/getsentry/sentry/blob/88d3b9eedda90f6d710a431ad841220aa0f35f56/src/sentry/stacktraces/platform.py
 
https://github.com/getsentry/relay/blob/a92fc6a6f5538c74a9344f9bd458bcbc93306ccb/relay-event-schema/src/protocol/constants.rs#L2